prefect_dbt/cli/configs/postgres.py

```diff
@@ -0,0 +1,75 @@
+"""Module containing models for Postgres configs"""
+from typing import Any, Dict
+
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal
+
+from prefect_dbt.cli.configs.base import MissingExtrasRequireError, TargetConfigs
+
+try:
+    from prefect_sqlalchemy.credentials import DatabaseCredentials
+except ModuleNotFoundError as e:
+    raise MissingExtrasRequireError("Postgres") from e
+
+
+class PostgresTargetConfigs(TargetConfigs):
```

Comment:
Can we add `_block_type_name`, `_logo_url`, `description`, and a code example to this block?